src/main/java/io/jenkins/plugins/designlibrary/Home.java

```diff
@@ -45,4 +47,15 @@ public UISample getDynamic(String name) {
         }
         return null;
     }
+
+    public String getPluginVersion() {
+        Jenkins jenkins = Jenkins.get();
+        if (jenkins != null) {
+            PluginWrapper plugin = jenkins.getPluginManager().getPlugin("design-library");
+            if (plugin != null) {
```

Comment:
Can `plugin` ever be null?

Reply:
If I dont use this if condition , it returns me:
Medium: Possible null pointer dereference in io.jenkins.plugins.designlibrary.Home.getPluginVersion()